src/main/resources/portal.properties.EXAMPLE

```diff
@@ -327,3 +327,6 @@ ehcache.cache_type=none
 
 # Enable gsva to query
 # skin.show_gsva=true
+
+# Installation map URL for public site
```

Comment:
This is a bit misleading - private portals could create their own installation map if they wanted. How about 

>Installation map URL for installation map iframes. If blank or commented out, no map will be shown and that page will be hidden.`